### PR TITLE
 freebsd fuse for mount for #649

### PR DESCRIPTION
**Pull request by jungle-boogie**

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -163,6 +163,23 @@ FUSE for OS X, which is available as a pre-release_.
...
+
+    $ sudo pkg install fusefs-libs
+    $ sudo kldload fuse
+    $ borg mount /mnt/backup::root-2013-08-02 /tmp/mymountpoint
+    $ ls /tmp/mymountpoint
+    bin  boot  etc  lib  lib64  mnt  opt  root  sbin  srv  usr  var
+    $ umount /tmp/mymountpoint
+
+
+
```
Comment (ThomasWaldmann) [inline docs/installation.rst]: I needed some more stuff to get it installed on freebsd:
https://github.com/borgbackup/borg/blob/master/Vagrantfile#L75

I am not a freebsd user, i just got it working on vagrant for testing.
Some of that stuff is only needed for vagrant or building python and should not be included here.
But some of the other stuff seems missing here (e.g. one does not want to kldload all the time after each reboot).

Also, do not give a mount / umount example here, that is not related to installation.

Comment (jungle-boogie) [inline docs/installation.rst]: > I needed some more stuff to get it installed on freebsd:

But that's already covered here:
https://github.com/borgbackup/borg/blob/master/docs/installation.rst#dependencies

 > e.g. one does not want to kldload all the time after each reboot).

Okay, I can change that.

> Also, do not give a mount / umount example here, that is not related to installation.

fair enough.

Comment (jungle-boogie) [inline docs/installation.rst]: What's bash used for?
https://github.com/borgbackup/borg/blob/master/Vagrantfile#L79

a build or run time dependency of what?

Comment (ThomasWaldmann) [inline docs/installation.rst]: the dependency installation is already described in general (for everybody) and specifically for some systems like debian/ubuntu, fedora, etc.

so, if we want to have more precise instructions for freebsd also, a section like the one seen for debian could be added with specific instructions.

Comment (ThomasWaldmann) [inline docs/installation.rst]: you can ignore the bash stuff in the Vagrantfile.

I just needed one same shell on every system to have the same scripts work on every platform.

**Pull request merged**